tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -107,7 +107,7 @@
       // The endpoint that serves trace viewer app.
       traceViewerBaseUrl: {
         type: String,
-        value: () => getRouter().pluginRoute("profile", "/trace_viewer"),
+        value: "/trace_viewer_index.html",
```

Comment:
I think this is correct. The URL of the trace viewer app is constant. Previously (with internal TensorBoard), the trace viewer html was served separately by the plugin, but the server is now serving all assets including "/index.html" and "/trace_viewer_index.html"(https://github.com/tensorflow/tensorboard/blob/master/tensorboard/BUILD#L57).